tests/security/389ds/tls_389ds_server.pm

```diff
@@ -0,0 +1,119 @@
+# Copyright (C) 2021 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: Implement & Integrate 389ds + sssd test case into openQA,
+#          This test module covers the server setup processes
+#
+# Maintainer: rfan1 <richard.fan@suse.com>
+# Tags: poo#88513, tc#1768672
+
+use base 'consoletest';
+use testapi;
+use strict;
+use warnings;
+use utils;
+use lockapi;
+use mmapi 'wait_for_children';
+use opensslca;
+
+sub run {
+    select_console("root-console");
```

Comment:
```suggestion
    $self->select_serial_terminal;
```

Reply:
Will do

Reply:
https://openqa.suse.de/tests/5506796

Seems new issue introduced if I used serial console. debugging

Reply:
https://progress.opensuse.org/issues/89011 was filed against the issue, then for now, I will use 'root-console' as a workaround